src/main/web-portal/tailwind.config.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
     extend: {
       backgroundImage: {
         'survival-1': "url('../public/survival-1.png')",
+        'survival-4': "url('../public/survival-4.png')",
```

Comment:
Hmmm... I know these pictures are kind of generic... but maybe we should name them accoording to what they depict?
Like idk: `survivor-looking-out-of-window-at-burning-plane`
What do you think?